.github/workflows/autoupdate.yml

```diff
@@ -216,6 +216,8 @@ jobs:
                 echo "Creating a PR..."
                 gh pr create --base "${{ matrix.upstream_repo_branch }}" --head "planemo-autoupdate:$REPO" --title "$TITLE" --repo "${{ matrix.upstream_repo_owner}}/${{ matrix.upstream_repo_name }}" --body-file "${{ github.workspace }}/body.txt"
               fi
+              # Go back to the main branch
+              git checkout "${{ matrix.upstream_repo_branch }}"
```

Comment:
It seems that clean up is scattered and duplicated in different places, should we instead do:

```shell
git checkout -- .
git checkout "${{ matrix.upstream_repo_branch }}"
```

after `cd "$REPO"` at line 93 and remove the other instances from the for loop?

Reply:
@nsoranzo you meant before `cd $REPO` no because if you do after it will not clean uncommitted changes done in another directory.